widgets/vss-extension.json

```diff
@@ -130,6 +130,36 @@
                     "supportedScopes": ["project_team"]
                 }
             },
+			{
+                "id": "Widget-for-sample-configuration",
+                "type": "ms.vss-dashboards-web.widget",
+                "targets": [
+                    "ms.vss-dashboards-web.widget-catalog", "Fabrikam.vsts-extensions-myExtensions.Sample-Configuration"
```

Comment:
No need to fully-qualify ... if contributing within the same manifest use ".xxx" .. so: ".Sample-Configuration"